test/test-v8debugapi.js

```diff
@@ -700,6 +700,31 @@ describe('v8debugapi', function() {
         process.nextTick(function() {foo();});
       });
     });
+
+    it('should not silence errors thrown in the wait callback', function(done) {
+      var message = 'This exception should not be silenced';
+      // Remove the mocha listener.
+      var originalListener = process.listeners('uncaughtException').pop();
```

Comment:
process.listeners returns a copy. Can't find the source quicky, but see here:
```
> var A = process.listeners('unhandledException')
undefined
> var B = process.listeners('unhandledException')
undefined
> A
[ [Function] ]
> B
[ [Function] ]
> A.pop()
[Function]
> A
[]
> B
[ [Function] ]
```

If there are multiple listeners, and the remaining ones don't abort, the test still works. If the remaining listeners abort, then that's fine – the test doesn't pass. But I agree that it would be simpler to add a test for single listener.